fastlane/spec/actions_specs/git_commit_spec.rb

```diff
@@ -54,6 +54,13 @@
       end
 
       it "generates the correct git command when configured to allow nothing to commit and there are changes to commit" do
+        allow(Fastlane::Actions).to receive(:sh)
+          .with("git status ./fastlane/README.md --porcelain")
+          .and_return("M  ./fastlane/README.md")
+        allow(Fastlane::Actions).to receive(:sh)
```

Comment:
```suggestion
        expect(Fastlane::Actions).to receive(:sh)
```

It feels like this must be called in the given scenario so `expect` makes sure future change will also behave the same. Though it is checked by below assertion, this would produce a slightly better error message when failing.

Reply:
You are right, I made the change